### DISCUSSION TITLE
WIP Chapters

### DISCUSSION THREAD

**Discussion by johko** [huggingface/computer-vision-course#184] [Ideas]
We finished our December sprint and got a lot of amazing content. :rocket: 
 
But there is still some content missing, of which some might be abandoned but some might also be Work in Progress.

If you are still working on something, please leave a reply here and tell us what you are working on and how far the progress is.

Thank you :slightly_smiling_face:

**Comment by ash-01xor** [comment]
Im currently leading the group for General Architecture for CNN , we are working on it and we will push it pretty soon
- Currently working on adding code and images for respective explanations.

**Comment by bellabf** [comment]
Hi, there was someone on my image and imaging team working on a notebook for data augmentation (mfidabel on discord). It was a bonus for the chapter.

**Comment by adhiiisetiawan** [comment]
Hi, i'm working on _**Model Optimization**_ chapter, currently still on going to create notebook for tools model optimization (TensorRT, openVino, onnx, etc) collaborate with @haizadtarik . All of notebook has done except on edgeTPU. So in general i'm cover about edgeTPU and make notebook to optimize model using the tools. But, currently still have an error about optimization part. Need to debug more deeply.

**Comment by Zekrom-7780** [comment]
Hello, I'm woking on _fine-tuning using BLIP (Image-to-Text)_ in the **Unit 4. Multimodal models: Transfer Learning**, and also in **Unit 12. Ethics and Biases**. The Notebook for Unit 4 is almost ready, but Google Colab's free space is causing the problem, and in Unit 12, I'm writing the .mdx file, for an explanation of a case study.

I was not able to work for the entriety of January, and will start on the 20th again

**Comment by mattmdjaga** [comment]
The Unit 4 - Multimodal Models - CLIP and relatives team is working on mdx files for:
Donut  (Document Analysis)
OneFormer (Segmentation) 
Non contrastive
Triplet

Each file has an assigned person who completed a different file so they all should be PRed this or next weerk.

**Comment by mmhamdy** [comment]
Hello everyone 👋
I'm working on Unit 11 (Zero Shot Computer Vision), and Unit 13 (Outlook and Trends). So, here is the status of each one:

- **Unit 11**
Active members: @mmhamdy, @albertkao227 
**Sections:**
Introduction --> ✅
Zero-shot Learning --> ✅
Zero-shot Image Recognition --> WIP
Zero-shot Object Detection --> was assigned but is now inactive
Zero-shot Image Segmentation --> unassigned
Challenges, Frontiers, and summary --> WIP

- **Unit 13**
Active members: @mmhamdy, @lulmer, @farrosalferro, @dariowsz, @gonzachiar
**Sections:**
Introduction --> WIP
Hiera --> ✅
Hyena --> ✅
I-JEPA --> ✅
RMT --> ✅ 
Trends And Research Directions --> WIP

**Comment by sezan92** [comment]
Hello everyone. We are still working on 

 - Unit 2
 - Active members: @albertkao227 @JvThunder @sitamgithub-MSIT @murtazaAmir @ShamieCC
 - Convolutional Neural Network: pretrained model: Googlenet, Resnext, Convnext
